### ISSUE TITLE
I got flagged on my main account this is the new one

### ISSUE THREAD

**Issue by Deus-Scissuras** [Deus-Scissuras/Unblocked-links-reborn#1]

**Comment by Deus-Scissuras** [comment]
@trickjoker312

**Comment by TrickJoker312** [comment]
Wait what
How did you get flagged?

**Comment by Deus-Scissuras** [comment]
idk they didnt state

**Comment by TrickJoker312** [comment]
Oh.

**Comment by Deus-Scissuras** [comment]
ty

**Comment by NotLexer9** [comment]
Also you got banned cause i had to report you for hacking and scamming.

**Comment by NotLexer9** [comment]
on my SuperLexer9 acc, also you are no hacker just scammer

**Comment by Deus-Scissuras** [comment]
how did i scammed you  i doxxed  you

**Comment by Deus-Scissuras** [comment]
im getting the report reverted this is just for now

**Comment by Deus-Scissuras** [comment]
i'l ip logged you  goofy @NotLexer9

**Comment by NotLexer9** [comment]
you know, I saw a guy who said you are scammer not hacker

**Comment by NotLexer9** [comment]
and yeah I had to lose my 2 accounts just so you can get banned cause what you did wasn't good.

**Comment by NotLexer9** [comment]
And as I previously said: I am going to leave you alone now

**Comment by NotLexer9** [comment]
I have VPN on

**Comment by NotLexer9** [comment]
plus I don't find a reason you have to ruin my life, cause well I'm new McProgrammerX and I don't wanna start drama cuz I can't keep making accounts I am soon going to run out of email addresses

**Comment by Deus-Scissuras** [comment]
i already got your ip

**Comment by Deus-Scissuras** [comment]
thats my goal

**Comment by NotLexer9** [comment]
Even if you get my IP it is VPN

**Comment by Deus-Scissuras** [comment]
ok bet first four digits are 73.12

**Comment by NotLexer9** [comment]
nope

**Comment by Deus-Scissuras** [comment]
i got it before remember even before your vpn

**Comment by Deus-Scissuras** [comment]
cap

**Comment by Deus-Scissuras** [comment]
Im   gonna learn to code for  get back

**Comment by NotLexer9** [comment]
So what is my IP address

**Comment by Deus-Scissuras** [comment]
i cant say here  or i'll get banned again

**Comment by Deus-Scissuras** [comment]
<code class="notranslate">&lt;script&gt;&lt;/script&gt;</code> i dont get how

**Comment by Deus-Scissuras** [comment]
you called me a kid 43 times and your deadass younger then me

**Comment by NotLexer9** [comment]
you know GitHub allows you to use HTML (cause it is a coding website like):
<h1>Hello world</h1>

I just typed in:
<plaintext><h1>Hello World</h1>

**Comment by Deus-Scissuras** [comment]
ok thanks

**Comment by NotLexer9** [comment]
and also we are all kids, cause well we are under the age of 18

**Comment by NotLexer9** [comment]
I already finished school last month

**Comment by Deus-Scissuras** [comment]
nice

**Comment by Deus-Scissuras** [comment]
<plaintext>

SUS

**Comment by Deus-Scissuras** [comment]
it didnt work

**Comment by Deus-Scissuras** [comment]
<plaintext> sus

**Comment by NotLexer9** [comment]
<"h1">Hello World</"h1">

something like that also remove the ""

**Comment by Deus-Scissuras** [comment]
<h1>Suss</h1>

**Comment by Deus-Scissuras** [comment]
ty

**Comment by NotLexer9** [comment]
yep

**Comment by NotLexer9** [comment]
also you can learn HTML here:
https://www.w3schools.com/html/

**Comment by Deus-Scissuras** [comment]
ty

**Comment by Deus-Scissuras** [comment]
<h1>𝕽𝖊𝖆𝖑𝖍𝖆𝖈𝖐1 𝖜𝖎𝖑𝖑 𝖗𝖊𝖙𝖚𝖗𝖓</h1>

**Comment by NotLexer9** [comment]
no problem also you can learn java here!:
https://w3schools.com/java

also I don't know Java but I know HTML

**Comment by Deus-Scissuras** [comment]
alr see ya

**Comment by NotLexer9** [comment]
k

**Comment by TrickJoker312** [comment]
This conversation could've happened in a new issue.

**Comment by Deus-Scissuras** [comment]
i know he decided this not me

**Comment by Deus-Scissuras** [comment]
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Glitch Text</title>
    <style>
        .glitch-text {
            font-size: 48px;
            font-weight: bold;
            text-transform: uppercase;
            color: #00ff00;
            text-shadow: 2px 2px 2px #00ff00, 4px 4px 4px #00ff00, 6px 6px 6px #00ff00;
            -webkit-animation: glitch 1.5s infinite alternate;
            animation: glitch 1.5s infinite alternate;
        }

        @keyframes glitch {
            0% { opacity: 1; }
            20% { opacity: 0; }
            40% { opacity: 1; }
            60% { opacity: 0; }
            80% { opacity: 1; }
            100% { opacity: 0; }
        }
    </style>
</head>
<body>
    <h1 class="glitch-text">Realhack1</h1>
</body>
</html>

**Comment by NotLexer9** [comment]
....
